pagopa/io-web-profile
# [IOPID-1020] Add RelayState and Intesi Group

## Short description

As per the title

## List of changes proposed in this pull request

- src/app/[locale]/_component/selectIdp/idpList.tsx: added RelayState param
- src/app/[locale]/_utils/idps.ts: added Intesi to idp list

## How to test

Try to login and verify that there is "Intesi Group" in the idp list and that the relay state parameter is added in the login url
